tests/test_commands.py

```diff
@@ -0,0 +1,47 @@
+import subprocess
+import sys
+import tempfile
+from pathlib import Path
+
+from web_poet.testing import Fixture
+
+
+def call_scrapy_command(cwd: str, *args: str) -> None:
+    with tempfile.TemporaryFile() as out:
+        args = (sys.executable, "-m", "scrapy.cmdline") + args
+        subprocess.call(args, stdout=out, stderr=out, cwd=cwd)
+
+
+def test_savefixture(tmp_path) -> None:
+    project_name = "foo"
+    cwd = Path(tmp_path)
+    call_scrapy_command(str(cwd), "startproject", project_name)
+    cwd /= project_name
+    type_name = "foo.po.BTSBookPage"
+    (cwd / project_name / "po.py").write_text(
+        """
+import attr
```

Comment:
```suggestion
import attrs
```

Let's have the tests use the new API.